controllers/volumereplicationgroup_controller.go

```diff
@@ -51,6 +51,10 @@ type VolumeReplicationGroupReconciler struct {
 	Scheme *runtime.Scheme
 }
 
+const (
+	vrgFinalizer = "volumereplicationgroup.storage.io"
```

Comment:
For improved readability in places where it is being used, to clarify that this is merely the name of the finalizer, can we add a "Name" suffix, vrgFinalizerName?

Reply:
ACK